gobblin-core/src/main/java/org/apache/gobblin/source/extractor/extract/restapi/RestApiConnector.java

```diff
@@ -77,6 +82,15 @@ public RestApiConnector(State state) {
         state.getPropAsInt(ConfigurationKeys.SOURCE_CONN_TIMEOUT, ConfigurationKeys.DEFAULT_CONN_TIMEOUT);
   }
 
+  @Override
+  public void close() throws IOException {
```

Comment:
When does this close() get called for Dynamics?

Reply:
For all restApiExtractor which may contain this connect, we expect the closeConnection method will close the connector. Will update PR to include this